src/components/Dashboard/DashboardComponent.jsx

```diff
@@ -107,27 +103,42 @@ class DashboardComponent extends Component {
               </Grid.Row>
             </Grid>
           </div>
-          <div id="art-charts">
-            <Grid>
-              <Grid.Row columns={2}>
-                <Grid.Column>
-                  <div id="area-chart">
-                    <Chart1 />
-                  </div>
-                </Grid.Column>
-                <Grid.Column>
-                  <Grid id="right-chart-grid">
-                    <Grid.Row columns={2} style={{ border: 'solid 1px #707070', borderRadius: '10px' }}>
-                      <Grid.Column>
-                        <Chart2 />
-                      </Grid.Column>
-                      <Grid.Column>
-                        <Chart3 />
-                      </Grid.Column>
-                    </Grid.Row>
-                  </Grid>
-                </Grid.Column>
-              </Grid.Row>
+
+          <Divider />
+
+          <div id="asset-links-section">
+            <Grid columns={4} stackable>
+              <Grid.Column>
+                <Link to="/assets">Asset List</Link>
+              </Grid.Column>
+
+              <Grid.Column>
+                <Link to="/asset-makes">Asset Makes</Link>
```

Comment:
Instead of enumerating all the asset links, why don't you make an array that you can map to the columns? By doing this, you don't have to copy-paste the columns each time we add more links to the section. E.g.

```
const assetNavigation = [
  {
     url: '/asset-makes',
     title:  'Asset Makes'
  },
  ...
];

class Dashboard extends Component {
  render() {
    return (
      ...
      {assetNavigation.map(nav => (
        <Grid.Column>
          <Link to={nav.url}>{nav.title}</Link>
        </Grid.Column>
      ))}
    );
  }
}